pepys_import/file/replay_comment_importer.py

```diff
@@ -0,0 +1,105 @@
+import os
+
+from .importer import Importer
+from pepys_import.core.validators import constants
+from pepys_import.core.formats.rep_line import parse_timestamp
+
+
+class ReplayCommentImporter(Importer):
+    def __init__(
+        self,
+        name="Replay Comment Importer",
+        validation_level=constants.ENHANCED_LEVEL,
+        short_name="REP Comment Importer",
+        separator=" ",
+    ):
+        super().__init__(name, validation_level, short_name)
+        self.separator = separator
+        self.text_label = None
+        self.depth = 0.0
+        self.errors = list()
+
+    def can_load_this_type(self, suffix):
+        return suffix.upper() == ".REP"
+
+    def can_load_this_filename(self, filename):
+        return True
+
+    def can_load_this_header(self, first_line):
+        return True
+
+    def can_load_this_file(self, file_contents):
+        return True
+
+    def load_this_file(self, data_store, path, file_contents, datafile):
+        self.errors = list()
+        basename = os.path.basename(path)
+        print(f"Rep Comment parser working on '{basename}'")
+        error_type = self.short_name + f" - Parsing error on '{basename}'"
+        datafile.measurements[self.short_name] = list()
+        for line_number, line in enumerate(file_contents, 1):
+            if line.startswith(";"):
+                if line.startswith(";NARRATIVE:"):
+                    # ok for for it
+                    tokens = line.split(" ")
```

Comment:
This might cause some problems if there are redundant whitespaces. For **split()** method, it is not the same with split() and split(" "). I remember that I had problems when I used split(" ") in REP Importer. Please check the following link: https://docs.python.org/3.7/library/stdtypes.html#str.split

> If sep is not specified or is None, a different splitting algorithm is applied: runs of consecutive whitespace are regarded as a single separator, and the result will contain no empty strings at the start or end if the string has leading or trailing whitespace. Consequently, splitting an empty string or a string consisting of just whitespace with a None separator returns [].

Reply:
Great - thanks. I've modified the sample data to include spaces.  It broke the test, and when I switched to `split()`, all worked fine :-)

Reply:
Okay, great 👍